tests/jballerina-unit-test/src/test/resources/test-src/statements/matchstmt/structured_error_match_patterns.bal

```diff
@@ -203,6 +203,7 @@ function foo5(any|error e) returns string {
             }
         }
     }
+    return "Not-matched";
```

Comment:
Is this correct? var x will match the error value as well. So every brach is returning. Hence this is unreachable code.